src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationHelper.cs

```diff
@@ -37,5 +47,25 @@ public class LinkedInAuthenticationHelper {
         /// Gets the public profile URL corresponding to the authenticated user.
         /// </summary>
         public static string GetPublicProfileUrl([NotNull] JObject user) => user.Value<string>("publicProfileUrl");
+
+        /// <summary>
+        /// Gets the industry corresponding to the authenticated user.
+        /// </summary>
+        public static string GetIndustry([NotNull] JObject user) => user.Value<string>("industry");
+
+        /// <summary>
+        /// Gets the summary corresponding to the authenticated user.
+        /// </summary>
+        public static string GetSummary([NotNull] JObject user) => user.Value<string>("summary");
+
+        /// <summary>
+        /// Gets the headline corresponding to the authenticated user.
+        /// </summary>
+        public static string GetHeadline([NotNull] JObject user) => user.Value<string>("headline");
+
+        /// <summary>
+        /// Gets the positions corresponding to the authenticated user.
+        /// </summary>
+        public static string GetPositions([NotNull] JObject user) => user["positions"].ToString();
```

Comment:
You can override the `OnCreatingTicket` event to add more claims (e.g http://stackoverflow.com/a/35634697/542757)